src/sql/workbench/contrib/backup/browser/backup.component.ts

```diff
@@ -232,6 +250,14 @@ export class BackupComponent extends AngularDisposable {
 			ariaLabel: LocalizedStrings.COPY_ONLY
 		});
 
+		// Set to url check box
+		this.toUrlCheckBox = new Checkbox(this.toUrlElement!.nativeElement, {
```

Comment:
These checkboxes should be registered too (while you're here if you could fix up the rest in this dialog that'd be appreciated)